### ISSUE TITLE
Does not work on SteamOS

### ISSUE THREAD

**Issue by ewmailing** [boppreh/steamgrid#58]
I'm trying to use the prebuilt steamgrid Linux binary on SteamOS. I have not had any success in seeing any downloaded or cached images. This is the output of the run:

`
steam@steamos:~/steamgrid$ ./steamgrid 
Loading overlays...
Loaded 6 overlays. 

You can find many user-created overlays at https://wwww.reddit.com/r/steamgrid/wiki/overlays .

Looking for Steam directory...
Loading users...
Setting permission...
Loading games for ewmailing
Loading existing images and backups...

0 images downloaded and 0 overlays applied.

Open Steam in grid view to see the results!

Press enter to close.
`

A couple of additional notes:

The normal user for a shell on SteamOS is 'desktop'. But the user account that 'owns' all the Steam games and userdata files is the user 'steam'. Running steamgrid as the desktop user can't access the directories, so steamgrid normally fails earlier. 

By default, 'steam' is not meant to be an account you can shell into, but you are allowed to set a password and 'su steam' which is what I did in the above example.

I have several dozens of games installed on SteamOS, so the "0 images downloaded" strikes me as something that has gone wrong.

Appreciate any help you can offer for this!

**Comment by Lucki** [comment]
Quick shot in the dark: Are your games publicly visible?

**Comment by boppreh** [comment]
Can you post the output of `ls -lR STEAMDIR/userdata/`, so that I can see the permissions and user/group ownership? Feel free to omit long entries like screenshots. Also, if you don't mind, can you try the [older release](https://github.com/boppreh/steamgrid/releases/tag/v2.1.1) to make sure it's not a regression?

**Comment by ewmailing** [comment]
@Lucki Great shot. That was indeed the problem. My profile was public, but my Games details were 'Friends only'. I changed it to public, and steamgrid then worked.

@boppreh Just for completeness, I originally was using the previous version (date stamp on my zip was May 25, 2017). I downloaded it a day or two ago. After I changed my profile, I went right to the new version. All seems to be working now.

Thank you both
